pom.xml

```diff
@@ -65,6 +66,29 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-resteasy-jackson</artifactId>
     </dependency>
+    <dependency>
+      <scope>test</scope>
```

Comment:
Just a nit here and below ... we usually put the scope as the last field (see e.g. https://github.com/strimzi/strimzi-kafka-operator/blob/fd6c08e6c923af6e38f1d3b6335a4493b09a8d79/pom.xml#L319). I think following the same would make it more readable since people are used to look there.

Reply:
Yeah, that makes sense. Thanks :)